server/Arcadia.Assistant.Calendar.Notifications/TemplateExpressionParser.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Arcadia.Assistant.Calendar.Notifications
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    public static class TemplateExpressionParser
+    {
+        private static readonly Regex ifBlockRegex = new Regex("(?<startIfGroup>{if:(?<startIfToken>.*?)}).*?(?<endIfGroup>{endif:(?<endIfToken>.*?)})");
+
+        public static string ParseTemplateExpression(this string template, IDictionary<string, string> context)
```

Comment:
so you apply it to ANY string.

Reply:
And what do you suggest? Use it like `new TemplateExpressionParser().Parse(...)`?

Reply:
yes

Reply:
Ugly.

Reply:
no. properly.

Reply:
Done.